constants/chainIds.json

```diff
@@ -31,6 +31,7 @@
   "250": "fantom",
   "269": "hpb",
   "288": "boba",
+  "314": "filecoin",
```

Comment:
that's 6 chains?

Reply:
afaik there should be a single chain here, if the other ones are testnets we shouldn't tag them as filecoin

Reply:
only mainnet

Reply:
Yes, the others are testnets.

Thank you for adding the mainnet one to  chainlist.org!